components/cli/pkg/util/utils.go

```diff
@@ -482,13 +482,11 @@ func UserHomeDir() string {
 
 func CelleryInstallationDir() string {
 	celleryHome := os.Getenv(constants.CELLERY_HOME_ENV_VAR)
-	if celleryHome == "" {
-		if runtime.GOOS == "darwin" {
-			celleryHome = constants.CELLERY_INSTALLATION_PATH_MAC
-		}
-		if runtime.GOOS == "linux" {
-			celleryHome = constants.CELLERY_INSTALLATION_PATH_UBUNTU
-		}
+	if runtime.GOOS == "darwin" {
+		celleryHome = constants.CELLERY_INSTALLATION_PATH_MAC
+	}
+	if runtime.GOOS == "linux" {
+		celleryHome = constants.CELLERY_INSTALLATION_PATH_UBUNTU
```

Comment:
This was an intentional change to allow using the CELLERY_HOME env var to override the default inferring of the installation directory. Shouldn't we have it ?

Reply:
Reverted to previous logic.